app/views/workflows/provide_metadata.html.erb

```diff
@@ -61,6 +61,8 @@
 
     <%= render partial: 'assets/controlled_vocab_annotations_form_properties', locals: { resource: @workflow } %>
 
+    <%= render partial: 'assets/controlled_vocab_annotations_form_property', locals: { resource: @workflow, property: :disciplines} -%>
```

Comment:
could this be included in `assets/controlled_vocab_annotations_form_properties` for consistency and so they are always grouped together